package.json

```diff
@@ -3,7 +3,7 @@
   "version": "4.0.0",
   "private": true,
   "scripts": {
-    "dev": "next dev -p ${CADENCE_WEB_PORT}",
+    "dev": "next dev -p ${CADENCE_WEB_PORT} | pino-pretty",
```

Comment:
Lets make sure this is our only option for `pino-pretty` as having it in code can make it more programatically configurable without having to modify both configs and scrpits

Reply:
This is the recommendation from the pino-pretty library: 
https://github.com/pinojs/pino-pretty/?tab=readme-ov-file#usage
https://github.com/pinojs/pino-pretty/?tab=readme-ov-file#programmatic-integration